tests/lms/views/error_test.py

```diff
@@ -3,87 +3,80 @@
 from pyramid.httpexceptions import HTTPNotImplemented
 import pytest
 
-from lms.views.error import ErrorViews
+from lms.views import error
```

Comment:
The tests for the error views haven't changed either, they've just been adapted now that the error views are functions not class methods.